tests/send-to-helix-workload-tests.proj

```diff
@@ -2,6 +2,9 @@
 
```

Comment:
Is this file really a `.proj` file? Since it is imported by the `send-to-helix.proj`, isn't it more of a `.targets` file?